torch/_dynamo/config.py

```diff
@@ -68,6 +69,18 @@
 # see [Note - on the state of mark_dynamic]
 assume_static_by_default = False
 
+
+class DYNAMIC_SHAPE_STRATEGY(Enum):
+    OFF = 1
+    # Find all tensors in frame and mark them all dynamic if they fail
+    ALL_FAILED_IN_FRAME = 2
+    # Flip to assume_static_by_default = False
+    # TODO: Do we want this? Seems useful... but also would kinda rather this be a bool.
+    ALL = 3
```

Comment:
I don't understand this enum entry

Reply:
The moment anything wobbles, it flips off assume_static_by_default and just runs as if we were fully dynamic.

Reply:
I can't really think of a situation where I would recommend this behavior to users. I'd axe it unless you can articulate a situation when someone should use this. Remember, someone has to explicitly opt into it, so if we don't tell anyone how to use the config flag, it might as well not exist.